### PR TITLE
Update the setup-gcloud GH action

### PR DESCRIPTION
**Pull request by tczekajlo**
**Proposed changes**:
- Update the setup-gcloud GH action due to security vulnerability

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by tczekajlo** [comment]
/modeltest

```yml

include:
 - dataset: ["Carbon Bot"]
   config: ["Sparse + DIET(bow) + ResponseSelector(bow)"]

```

File: .github/workflows/ci-model-regression-on-schedule.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
           ./gomplate -f .github/runner/github-runner-deployment.yaml.tmpl -o runner_deployment.yaml
 
       # Setup gcloud CLI
-      - uses: GoogleCloudPlatform/github-actions@0.1.2
+      - uses: GoogleCloudPlatform/github-actions@e23988b2af9696c66e87d1efbc688d3a80c3be14
         with:
```
Comment (m-vdb) [inline .github/workflows/ci-model-regression-on-schedule.yml:55]: do we want to set the version here as well or no?

**Review by m-vdb — APPROVED** [review]

File: .github/workflows/ci-model-regression-on-schedule.yml (hunk 1 above)
Comment (tczekajlo) [reply .github/workflows/ci-model-regression-on-schedule.yml:55]: No, the latest tagged version is tooo old that's why we use git sha :(

Comment (m-vdb) [reply .github/workflows/ci-model-regression-on-schedule.yml:55]: I meant a `version` key under `with:` 😄

Comment (tczekajlo) [reply .github/workflows/ci-model-regression-on-schedule.yml:55]: Aaaa, I see 😄  a good point!

**Pull request merged**